### PR TITLE
Test improvements

### PR DESCRIPTION
**Pull request by smoralesd**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000 **N/A**
- [x] Added relevant unit test for your changes. (`npm run test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`npm run precheckin`)
- [ ] Added screenshots/GIFs for UI changes.

#### Description of changes

Add missing tests.
Not using `export default` anymore as we have a tslint rule around this.

#### Notes for reviewers

**N/A**

### CODE REVIEW

File: src/tests/unit/tests/assessments/manual-test-record-your-results.test.tsx [hunk 1]
```diff
@@ -3,28 +3,15 @@
...
-            .at(1)
-            .getElement();
-        expect(li.props.children[1]).toBe('a failure');
-        expect(li.props.children[5]).toBe(' then add the failure instance');
-    });
+    const multiple = [false, true];
 
-    test('render: isMultipleFailurePossible = true', () => {
-        const props = { isMultipleFailurePossible: true };
+    it.each(multiple)('- render: isMultipleFailurePossible = %p', isMultipleFailurePossible => {
```
Comment (dbjorge) [inline src/tests/unit/tests/assessments/manual-test-record-your-results.test.tsx]: >-  [](start = 23, length = 2)

nit: why the extra punctuation?

File: src/tests/unit/tests/assessments/assisted-test-record-your-restuls.test.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import { AssistedTestRecordYourResults } from '../../../../assessments/common/assisted-test-record-your-results';
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```
Comment (dbjorge) [inline src/tests/unit/tests/assessments/assisted-test-record-your-restuls.test.tsx]: Move copyright banner to top?

File: src/tests/unit/tests/assessments/manual-test-record-your-results.test.tsx (hunk 1 above)
Comment (smoralesd) [reply src/tests/unit/tests/assessments/manual-test-record-your-results.test.tsx]: do you mean the '-'? forgot to remove it, it was so the "new" test didn't have the same name as the old ones... I'll update.

File: src/tests/unit/tests/assessments/assisted-test-record-your-restuls.test.tsx (hunk 2 above)
Comment (smoralesd) [reply src/tests/unit/tests/assessments/assisted-test-record-your-restuls.test.tsx]: Done

File: src/tests/unit/tests/assessments/assisted-test-record-your-restuls.test.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (peterdur) [inline src/tests/unit/tests/assessments/assisted-test-record-your-restuls.test.tsx]: typo in file name: assisted-test-record-your-**restuls**.test.tsx (should be **results**)

**Pull request closed without merging**